aten/src/ATen/native/vulkan/VulkanAten.cpp

```diff
@@ -262,8 +262,7 @@ at::Tensor vulkan_convolution_prepacked(
       VulkanTensor{{params.N, params.OC, params.OH, params.OW}};
   voutput.allocate_storage();
   const bool hasBias = bias.has_value() && bias->defined();
-  const bool vulkanBias = (*bias).is_vulkan();
-  if (hasBias && vulkanBias) {
+  if (hasBias && (*bias).is_vulkan()) {
```

Comment:
Alternatively you can write as
```
if (hasBias && bias->is_vulkan()) {
...
}
```
since the dereference operator is overloaded for c10::optional.  Not that it really matters but just maybe a tiny bit more idiomatic.